editor/src/components/canvas/dom-walker-caching.spec.browser2.tsx

```diff
@@ -124,17 +124,12 @@ describe('Dom-walker Caching', () => {
     ])
 
     expect(saveDomReportActions[2].invalidatedPaths).toEqual([
-      'storyboard-entity/scene-1-entity',
       'storyboard-entity',
       'storyboard-entity/scene-1-entity/app-entity:app-outer-div',
       'storyboard-entity/scene-1-entity/app-entity:app-outer-div/card-instance:card-outer-div',
       'storyboard-entity/scene-2-entity/same-file-app-entity:same-file-app-div',
     ])
-    expect(saveDomReportActions[2].cachedPaths).toEqual([
-      EP.fromString('storyboard-entity/scene-2-entity/same-file-app-entity:same-file-app-div'),
-      EP.fromString('storyboard-entity/scene-2-entity/same-file-app-entity'),
-      EP.fromString('storyboard-entity/scene-2-entity'),
-    ])
+    expect(saveDomReportActions[2].cachedPaths).toEqual([])
```

Comment:
This is because we invalidate the DOM walker cache as a whole when we detect a parsed model update coming from the parser-printer, so this is also the reason for the render count tests bump